lib/views/location.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:one_app/utils/location_time.dart';
+import 'package:flutter_spinkit/flutter_spinkit.dart';
+
+class Location extends StatefulWidget {
+  final LocationTime location;
+  const Location({super.key, required this.location});
+
+  @override
+  State<Location> createState() => _LocationState();
+}
+
+class _LocationState extends State<Location> {
+  void setUpTime() async {
+    await widget.location.getTime();
```

Comment:
This `getTime` API is a little strange. I always expect a method names "get" to return something, but this one doesn't. Maybe instead of storing the data internally to the LocationTime return it here as an object that this state can hold on to?